### PR TITLE
#664 TripSearchCriteria - only one of departure/arrival time may be p…

### PR DESCRIPTION
**Pull request by jspetrak**
…rovided

### CODE REVIEW

File: specification/v3.4/OSDM-online-api-v3.4.0-draft.yml [hunk 1]
```diff
@@ -14143,7 +14143,7 @@ components:
       properties:
         departureTime:
           description: |
-            Needs to in local date time format.
+            Needs to in local date time format. If departure time is provided the departure time must be omitted.
```
Comment (VelvetteRld) [inline specification/v3.4/OSDM-online-api-v3.4.0-draft.yml:14146]: Should be
"If departure time is provided the arrival time must be omitted." to mimic line 14152

File: specification/v4.0/OSDM-online-api-v4.0.0-draft.yml [hunk 2]
```diff
@@ -13997,7 +13997,7 @@ components:
       properties:
         departureTime:
           description: |
-            Needs to in local date time format.
+            Needs to in local date time format. If departure time is provided the departure time must be omitted.
```
Comment (VelvetteRld) [inline specification/v4.0/OSDM-online-api-v4.0.0-draft.yml:14000]: Should be
"If departure time is provided the arrival time must be omitted." to mimic line 14006

**Review by VelvetteRld — CHANGES_REQUESTED** [review]
Suggesting adjusting the added sentence to not repeat "departure time" twice

**Comment by jspetrak** [comment]
Descriptions fixed based on discussion from the Technical Group Call.

**Review by CGantert345 — APPROVED** [review]

**Pull request merged**